docs/src/environments.md

```diff
@@ -107,3 +112,146 @@ Otherwise, it will resolve the latest versions of the dependencies compatible wi
     ```bash
     $ julia --project=. myscript.jl
     ```
+
+
+## Temporary environments
+
+Temporary environments make it easy to start an environment from a blank slate to test a package or set of
+packages, and have Pkg automatically delete the environment when you're done.
+For instance, when writing a bug report, you may want to test your minimal reproducible
+example in a 'clean' environment to ensure it's actually reproducible as written. You might
+also want a scratch space to try out a new package, or a sandbox to resolve version conflicts
+between several incompatible packages.
+
+```julia-repl
+(@v1.6) pkg> activate --temp # requires Julia 1.5 or later
+  Activating new environment at `/var/folders/34/km3mmt5930gc4pzq1d08jvjw0000gn/T/jl_a31egx/Project.toml`
+
+(jl_a31egx) pkg> add Example
+    Updating registry at `~/.julia/registries/General`
+   Resolving package versions...
+    Updating `/private/var/folders/34/km3mmt5930gc4pzq1d08jvjw0000gn/T/jl_a31egx/Project.toml`
+  [7876af07] + Example v0.5.3
+    Updating `/private/var/folders/34/km3mmt5930gc4pzq1d08jvjw0000gn/T/jl_a31egx/Manifest.toml`
+  [7876af07] + Example v0.5.3
+```
+
+The temporary directory containing this environment will be removed when you exit Julia,
+so you don't need to worry about interference with your main package environment(s).
+
+Note that this still uses the global package depot (typically `~/.julia`), which is shared between environments.
+Perhaps when troubleshooting an especially thorny package management issue, you may want to ensure the
+temporary environment's package depot is completely independent from any other package environment by
+setting the [`JULIA_DEPOT_PATH`](https://docs.julialang.org/en/v1/manual/environment-variables/#JULIA_DEPOT_PATH) environment variable.
+For example,
+
+```console
+# Linux Bash
+root@dc76efc35971:/# JULIA_DEPOT_PATH=/tmp/juliatmp julia
+
+(@v1.6) pkg> st
+  Installing known registries into `/tmp/juliatmp`
+      Status `/tmp/juliatmp/environments/v1.6/Project.toml` (empty project)
+
+(@v1.6) pkg> add Example
+    Updating registry at `/tmp/juliatmp/registries/General.toml`
+   Resolving package versions...
+   Installed Example ─ v0.5.3
+    Updating `/tmp/juliatmp/environments/v1.6/Project.toml`
+  [7876af07] + Example v0.5.3
+    Updating `/tmp/juliatmp/environments/v1.6/Manifest.toml`
+  [7876af07] + Example v0.5.3
+Precompiling project...
+  1 dependency successfully precompiled in 0 seconds
+```
+
+This way, anything downloaded or modified by package operations will be stored in the temporary directory
+`/tmp/juliatmp`, with no need to worry that it might break your regular workflow.
+
+## [shared environment](@id shared-environment)
```

Comment:
I don't think this description is correct for shared environments. A shared environment is just an environment that can be referred to by name and is put in `~/.julia/environments/`. For example:

```
(@v1.7) pkg> activate --shared Foo
  Activating project at `~/.julia/environments/Foo`
```

This is completely separate from `LOAD_PATH` and a shared environment created like above will not be in the default load path if you activate a different project.